### ISSUE TITLE
False positives in *.spec.ts files

### ISSUE THREAD

**Issue by Plondrein** [takuya-nakayasu/eslint-plugin-angular-file-naming#1]
I got a lot of errors due to having some test components inside *.spec.ts files - is it possible  to skip those by default?

**Comment by takuya-nakayasu** [comment]
Thanks for your comment.

> I got a lot of errors due to having some test components inside *.spec.ts files - is it possible to skip those by default?

There is two way to skip *.spec.ts files.

1. Add ` "suffixes": []` to `"spec"`

```json
       // .eslintrc.json
      "rules": {
        "angular-file-naming/component-filename-suffix": [
          "error",
          {
            "suffixes": ["component", "page", "spec"]
          }
        ],
```

2. Creating a `.eslintignore` on your project root folder with the thing you want to ignore:

```
**/*.spec.ts
```

**Comment by Plondrein** [comment]
Thanks, I will try this out - I tried previously excluding 'component.spec'

**Comment by takuya-nakayasu** [comment]
v1.0.3 has been released.
In v1.0.3, `* spec.ts` files are excluded by default.

https://github.com/takuya-nakayasu/eslint-plugin-angular-file-naming/releases/tag/v1.0.3

**Comment by takuya-nakayasu** [comment]
This issue has been resolved in v1.0.3.
